### ISSUE TITLE
1.7 Editing  and Deleting a User

### ISSUE THREAD

**Issue by Fatoumatabintabarry** [Irisvella/InventoryPilot#101]
As an admin, I want to be able to edit user accounts  and delete a user when he leaves the company in order to manage access the dashboard.

Story points: 6
Priority: 1 (high)
Risk: Low
Child of: https://github.com/christa-ux/InventoryPilot/issues/8

**Tasks:**

- [x] Create a frontend interface with fields to fill (username, email, password, role, name, DOB). (1 points)
#34 
 - [x] Create a backend API endpoint for account update  (2 points)
 - [x] Create a backend API endpoint for account deletion (2 points)
 #36
 - [x] Trigger API call on form submission 
#37

**Comment by Fatoumatabintabarry** [comment]
completed

**Comment by yousefenein** [comment]
Referencing the UI update here : [c17701ed72a00ad49920301efb705c1680207739]

**Comment by yousefenein** [comment]
UI updates completed, just need to integrate new sidebar once it is merged into main
